src/controllers/FrameController.ts

```diff
@@ -1,13 +1,14 @@
-import type { EditorAPI, Id } from '../../types/CommonTypes';
-import { getCalculatedValue } from '../utils/getCalculatedValue';
-import { getEditorResponseData } from '../utils/EditorResponseData';
+import type {EditorAPI, Id} from '../../types/CommonTypes';
```

Comment:
Minor, but I think there is an issue with prettier config

Reply:
Indeed IIRC there is a manual step to enable automatic prettier in the commit hook. I re-ran it for this branch.